client/components/Main/CreateNewSubnavDropdown.tsx

```diff
@@ -63,7 +63,13 @@ class CreateNewSubnavDropdownFn extends React.PureComponent<IProps> {
             const sortedTemplates = eventTemplates
                 .sort((templ1, templ2) => templ1.template_name.localeCompare(templ2.template_name));
 
-            recentTemplates
+            /**
+             * If recent templates has empty then in this case show eventtemplates.
+             */
```

Comment:
I think the code is self explanatory here. No need for a comment

Reply:
okay